get-started/graph-database/graph-database.adoc

```diff
@@ -20,93 +20,89 @@ You should have a basic understanding of what a database is.
 
 toc::[]
 
-=== Overview
+== Overview
 
-Very simply, a graph database is a database designed to treat the relationships between data as a first-class citizen in the data model.  
+Very simply, a graph database is a database designed for storing and retrieving data and the relationships among it.
+Rather than prioritizing a rigidly-formatted structure to force on the data and navigating the structure to analyze sets of data, a graph database directly connects each piece of data to those it is related to, elevating the relationships in the data model to equal importance with the data itself.
```

Comment:
edit: "Rather than ~~prioritizing~~ **forcing** a rigidly-formatted structure ~~to force~~ on the data and navigating..."

Reply:
I would love a more natural sentence, and not something that feels so abstract/academic. After all neo4j was always about pragmatic solutions.